mindsdb_native/libs/backends/lightwood.py

```diff
@@ -84,6 +84,14 @@ def _create_timeseries_df(self, original_df):
 
                     group_by_ts_map[k].iloc[i][order_col].reverse()
 
+        if self.transaction.lmd['tss']['use_previous_target']:
+            for target_column in self.transaction.lmd['predict_columns']:
+                for k in group_by_ts_map:
+                    previous_target_values = list(group_by_ts_map[k][target_column])
+                    del previous_target_values[-1]
+                    previous_target_values = [None] + previous_target_values
```

Comment:
simplifies to `previous_target_values = [None] + group_by_ts_map[k][target_column][:-1]`
or `previous_target_values = [None, *group_by_ts_map[k][target_column][:-1]]`